hatServer/static/js/services/surveyQuestions.js

```diff
@@ -0,0 +1,461 @@
+    var firstName = {
+        included: true,
+        required: true
+    };
+    var lastName = {
+        included: true,
+        required: true
+    };
+    var preferredName = {
+        included: true,
+        required: false
+    };
+    var overallGPA = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var csGPA = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var skillCategories = {
+        expert: true,
+        good: true,
+        basic: true,
+        none: true
+    };
+    var overallProgramming = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var databaseDevelopment = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var embeddedSystems = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var webApplications = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var mobileApplications = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var userInterface = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var statistics = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var networking = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var security = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var robotics = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var computerVision = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var algorithms = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var machineLearning = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var firstChoice = {
+        included: true,
+        required: true,
+        weight: 2
+    };
+    var firstComment = {
+        included: true,
+        required: false
+    };
+    var secondChoice = {
+        included: true,
+        required: true,
+        weight: 2
+    };
+    var secondComment = {
+        included: true,
+        required: false
+    };
+    var thirdChoice = {
+        included: true,
+        required: true,
+        weight: 2
+    };
+    var thirdComment = {
+        included: true,
+        required: false
+    };
+    var fourthChoice = {
+        included: true,
+        required: true,
+        weight: 2
+    };
+    var fourthComment = {
+        included: true,
+        required: false
+    };
+    var fifthChoice = {
+        included: true,
+        required: true,
+        weight: 2
+    };
+    var fifthComment = {
+        included: true,
+        required: false
+    };
+    var ipPreference = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var ipOptions = {
```

Comment:
Variables for ip rights I used were RETAIN and NO_PREF if you want to keep in consistent, or I can change it in the algorithm.
I know 'Depends on the project' was an option, but it's kind of useless as far as the algorithm goes, so I was just going to make it NO_PREF as well